src/commands/functions/functions-create.js

```diff
@@ -565,6 +628,41 @@ const installAddons = async function (command, functionAddons, fnPath) {
   return Promise.all(arr)
 }
 
+const registerEFInToml = async (funcName) => {
+  if (!fs.existsSync('netlify.toml')) {
+    log(`${NETLIFYDEVLOG} \`netlify.toml\` file does not exist yet. Creating it...`)
+  }
+
+  let { funcPath } = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'funcPath',
+      message: `What route do you want your edge function to be invoked on? \nRoute can be configured by editing your \`netlify.toml\` file.`,
```

Comment:
Not a big deal, but I find the new-line here a bit unexpected. Since you're telling people that they can change the route after they select it, I'm not sure we need to also do it here. Up to you! If you want to keep it, I'd recommend getting rid of the `\n`.

```suggestion
      message: `What route do you want your edge function to be invoked on?`,
```